internal/identity/gitlab.go

```diff
@@ -7,6 +7,8 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/pomerium/pomerium/internal/log"
+
```

Comment:
nit: group pomerium based imports together

Reply:
`gp.UserGroupFn = gp.UserGroups`, I honestly thought I had it in there. Thanks for pointing this out, I'll implement the feedback